### ISSUE TITLE
Deleted Nodes constantly reappearing

### ISSUE THREAD

**Issue by Nistum** [MrManiacc/Blueprints#3]
Hi. After the latest update, the GUI now just reloads all nodes, including ones that were deleted. This isn't a huge issue per say on smaller layouts with few to no errors, but can and does get messy trying to sort through a lot of them as they overlap. This happens every single time the GUI is opened, even if you open it right after closing it.

**Comment by MrManiacc** [comment]
Yes I internally changed the way nodes are stored. They are now completely stored on the server, and the client gui requests them every time it wants to render the screen.
